src/assets/toolkit/styles/components/sky.css

```diff
@@ -37,7 +37,7 @@
 
 .Sky {
   position: relative; /* 1 */
-  color: var(--Sky-color);
+  color: var(--Sky-color-light);
```

Comment:
`--Sky-color` is used for the links on hover currently. I guess that is a good point though, if that is all it's used for it could probably have a different variable name.